<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>761</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="START">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EXIT">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IP">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>INPUT_IP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PORT">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>INPUT_PORT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PATH">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>INPUT_PATH</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PORT">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PATH">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_IP">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_PORT">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_PATH">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IP">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CLEAR">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>230</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CLEAR_WINDOW</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuweb_server_zyp">
    <property name="title">
     <string>web_server_zyp</string>
    </property>
   </widget>
   <addaction name="menuweb_server_zyp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
